src/clr-addons/slider/slider.scss

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (c) 2018 Porsche Informatik. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+@include exports('slider.clr-addons') {
+  // Slider
+  $clr-slider-size-sm: 75px;
+  $clr-slider-size-md: $clr-slider-size-sm * 2;
+  $clr-slider-size-lg: $clr-slider-size-sm * 3;
+  $clr-slider-size-xl: $clr-slider-size-sm * 4;
+
+  .slider-action-field {
+    float: left;
+  }
+
+  .value-boundary {
+    width: 10px;
+  }
+
+  .slider-wrapper {
+    display: inline-block;
+    width: 25px;
+    height: 25px;
+    padding: 0;
+  }
+
+  .slider-vertical {
+    height: 20px;
+    margin: 0;
+    transform: rotateZ(270deg);
+    -moz-transform: rotateZ(270deg);
+    -webkit-transform: rotateZ(270deg);
+    -ms-transform: rotateZ(270deg);
+  }
+
+  .slider-size-sm:not(.slider-wrapper) {
+    width: $clr-slider-size-sm;
+    transform-origin: $clr-slider-size-sm/2 $clr-slider-size-sm/2;
+  }
+  .slider-size-md:not(.slider-wrapper) {
+    width: $clr-slider-size-md;
+    transform-origin: $clr-slider-size-md/2 $clr-slider-size-md/2;
+  }
+  .slider-size-lg:not(.slider-wrapper) {
+    width: $clr-slider-size-lg;
+    transform-origin: $clr-slider-size-lg/2 $clr-slider-size-lg/2;
+  }
+  .slider-size-xl:not(.slider-wrapper) {
+    width: $clr-slider-size-xl;
+    transform-origin: $clr-slider-size-xl/2 $clr-slider-size-xl/2;
+  }
+
+  .slider-wrapper.slider-size-sm {
+    height: $clr-slider-size-sm;
+  }
+  .slider-wrapper.slider-size-md {
+    height: $clr-slider-size-md;
+  }
+  .slider-wrapper.slider-size-lg {
+    height: $clr-slider-size-lg;
+  }
+  .slider-wrapper.slider-size-xl {
+    height: $clr-slider-size-xl;
+  }
+
+  input[type='range'] {
+    background: transparent;
+    height: 25px;
+    -webkit-appearance: none;
+    margin: 10px 0;
+    width: 100%;
+  }
+  input[type='range']:focus {
+    outline: none;
```

Comment:
in scss you can write the following:

input[type=range] {
  input type css here;

  :focus {
    focus css here;
  }
}